conf/nomad/presto.hcl

```diff
@@ -45,6 +45,13 @@ job "${nomad_job_name}" {
             }
           }
         }
+        sidecar_task {
+          driver = "docker"
```

Comment:
It is not necessary to put `driver = docker`, it is default value [reference](https://www.nomadproject.io/docs/job-specification/sidecar_task#driver)
```suggestion
```